src/Microsoft.VisualStudio.Editor.Razor/DefaultRazorIndentationFactsService.cs

```diff
@@ -63,40 +59,17 @@ internal class DefaultRazorIndentationFactsService : RazorIndentationFactsServic
             }
 
             int? desiredIndentation = null;
-
             SyntaxTreeNode owningChild = owningSpan;
-            while ((owningChild.Parent != null) && !desiredIndentation.HasValue)
+            while (owningChild.Parent != null)
             {
                 var owningParent = owningChild.Parent;
-                var children = new List<SyntaxTreeNode>(owningParent.Children);
-                for (var i = 0; i < children.Count; i++)
+                for (var i = 0; i < owningParent.Children.Count; i++)
                 {
-                    var currentChild = children[i];
-                    if (!currentChild.IsBlock)
+                    var currentChild = owningParent.Children[i];
+                    if (IsCSharpOpenCurlyBrace(currentChild))
                     {
-                        var currentSpan = currentChild as Span;
-                        if (currentSpan.Symbols.Count == 1 &&
-                            currentSpan.Symbols[0] is CSharpSymbol symbol &&
-                            symbol.Type == CSharpSymbolType.LeftBrace)
-                        {
-                            var extraIndent = 0;
-
-                            // Dev11 337312: Only indent one level deeper if the item after the open curly brace is a markup block
```

Comment:
Wasn't being used and all their tests are passing so 👍

Reply:
What about our tests? was this truly dead code?

We shouldn't count on their tests to have detailed coverage of these cases.

Reply:
Truly dead code, `extraIndent` is never used

Reply:
Oh wow, ok - great cleanup then 💯 🍺 🎄 :canada: